session/manager.go

```diff
@@ -2,33 +2,51 @@ package session
 
 import (
 	"fmt"
+	"sync"
+	"time"
 
 	"github.com/goravel/framework/contracts/config"
 	"github.com/goravel/framework/contracts/foundation"
 	sessioncontract "github.com/goravel/framework/contracts/session"
 	"github.com/goravel/framework/session/driver"
+	"github.com/goravel/framework/support/color"
 )
 
 type Manager struct {
-	config        config.Config
-	customDrivers map[string]sessioncontract.Driver
-	drivers       map[string]sessioncontract.Driver
-	json          foundation.Json
+	config      config.Config
+	drivers     map[string]sessioncontract.Driver
+	json        foundation.Json
+	sessionPool sync.Pool
 }
 
 func NewManager(config config.Config, json foundation.Json) *Manager {
 	manager := &Manager{
-		config:        config,
-		customDrivers: make(map[string]sessioncontract.Driver),
-		drivers:       make(map[string]sessioncontract.Driver),
-		json:          json,
+		config:  config,
+		drivers: make(map[string]sessioncontract.Driver),
+		json:    json,
+		sessionPool: sync.Pool{New: func() any {
+			return &Session{
+				attributes: make(map[string]any),
+			}
+		},
+		},
 	}
-	manager.registerDrivers()
+	manager.extendDefaultDrivers()
 	return manager
 }
 
 func (m *Manager) BuildSession(handler sessioncontract.Driver, sessionID ...string) sessioncontract.Session {
-	return NewSession(m.config.GetString("session.cookie"), handler, m.json, sessionID...)
+	session := m.AcquireSession()
```

Comment:
We shouldn't directly access properties from the session. Could you please use methods instead of accessing session.name directly?